### PR TITLE
Stats test

### PR DESCRIPTION
**Pull request by gaaaach**
### Checklist
- [x] I've run `mvn test` from the root directory to see all new and existing tests pass
- [x] I've followed code style and run and ensured the code style is valid
- [x] I've created new tests if necessary.

### Motivation and Context
https://github.com/ctco-dev/feedback-mood-rate/issues/46

### Description
Made testus

### CODE REVIEW

File: src/main/java/lv/ctco/javaschool/vote/boundary/VoteApi.java [hunk 1]
```diff
@@ -181,11 +181,11 @@ public void mergeEventVote(EventVoteDto feedback) {
         List<DailyVote> dailyVotesList;
```
Comment (StepanovsRomans) [inline src/main/java/lv/ctco/javaschool/vote/boundary/VoteApi.java]: reformat

File: src/test/java/lv/ctco/javaschool/vote/boundary/VoteApiTest.java [hunk 2]
```diff
@@ -171,4 +175,119 @@ void checkTodayDate() {
         eventVote.setEvent(event);
         assertTrue(voteApi.checkTodayDate(eventVote));
     }
+
+
+    @Test
+    @DisplayName("Check if day statistics is selected then returned StatisticsDto of DailyVote class")
+    void getStatisticsForOneDay() {
+        LocalDate date = LocalDate.of(2018,10,10);
```
Comment (StepanovsRomans) [inline src/test/java/lv/ctco/javaschool/vote/boundary/VoteApiTest.java]: useless object LocalDate

File: src/main/java/lv/ctco/javaschool/vote/boundary/VoteApi.java [hunk 3]
```diff
@@ -181,11 +181,11 @@ public void mergeEventVote(EventVoteDto feedback) {
...
         List<StatisticsDto> statsList = new ArrayList<>();
 
-        if ( dateDto.getDate() != null) {
+        if ( dateDto.getDate() != null && dateDto.getWeek() == null) {
```
Comment (StepanovsRomans) [inline src/main/java/lv/ctco/javaschool/vote/boundary/VoteApi.java]: add test to check if dateDto will be null
